client/src/get_auction_info.rs

```diff
@@ -12,12 +12,13 @@ enum DisplayOrder {
     Verbose,
     NodeAddress,
     RpcId,
+    BlockIdentifier,
 }
 
 impl<'a, 'b> ClientCommand<'a, 'b> for GetAuctionInfo {
     const NAME: &'static str = "get-auction-info";
     const ABOUT: &'static str =
-        "Retrieves the bids and validators as of the most recently added block";
+        "Retrieves the bids and validators as of a block added to the linear chain";
```

Comment:
"Retrieves the bids and validators as of either a specific block (by height or hash), or the most recently added block";